buildpipeline/Core-Setup-PortableLinux-x64-master.json

```diff
@@ -0,0 +1,154 @@
+{
+  "build": [
+    {
```

Comment:
You should add a "Docker rm $(DockerContainerName)" step at the beginning of this build (as well as parameterizing the name of the container, dotnetcore-setup-build-container-rhel). We just ran into an issue where a RHEL build was killed before it could clean up its docker container, then the next RHEL build on that machine failed because it tried to create a container with the same name as the pre-existing container from the last job. See here - https://mseng.visualstudio.com/DefaultCollection/dotnetcore/_build/definitionEditor?definitionId=3545&_a=simple-process